src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
+import "./assets/css/styles.css";
+import App from "./components/App";
```

Comment:
`App` should live outside of components folder.

Reply:
Moved to `components/App.js`